backend/namespaces/dummy.py

```diff
@@ -50,7 +50,7 @@ def post(self):
         }
 
     @dummy.response(200, 'Success')
-    @dummy.response(400, 'Malformed Request')
+    @dummy.response(400, 'Malformed Request / Post Not Found')
```

Comment:
If the request is valid, but the post cannot be found I'd suggest, 404 is a better status code.

Reply:
I 100% agree. This is what I had at first, but flask_restplus decides to be helpful and suggests that the endpoint is wrong, which is why I decided to go with 400. I can change it to 404, but then you get this:

```json
{
    "message": "Post Not Found. You have requested this URI [/dummy/post] but did you mean /dummy/post or /dummy/post/like or /dummy/post/unlike ?"
}
```

Reply:
So it does include the message at first though. Maybe extend the message to be:

`Post Not Found: This was a valid request, but the database could not find the post you're looking for. Else, you may also have hit the wrong endpoint.`

Verbose, but will clarify the above a bit better.

I think the restplus would work if the route was post/<id> but it ain't. Oh wellz.

Reply:
Found a solution.

```python
app.config['ERROR_404_HELP'] = False
```

It gets rid of the message, but it does it for everything. Shouldn't be too much of a problem, right?